rules/web/web_java_in_access_log.yml

```diff
@@ -17,8 +17,7 @@ detection:
       - '${(#a=@'
       - '%24%7B%40java'
       - '${@java'
-      - '\u0022java'
-      - '"java'
+      - 'java.lang.'
```

Comment:
What happened to `\u0022java` ?

Reply:
Fix, the others must be significant of a java payload